.github/workflows/fork-sync.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  schedule:
+    - cron:  '0 7 * * 1,4'
+  workflow_dispatch:
+jobs:
+  sync_with_upstream:
+    runs-on: ubuntu-latest
+    name: Sync main with upstream latest
+    steps:
+    - name: Checkout stirling-labs branch
+      uses: actions/checkout@v2
+      with:
+        ref: stirling-labs
+
+    - name: Pull upstream changes
+      id: sync
+      uses: aormsby/Fork-Sync-With-Upstream-action@v2.3
+      with:
+        upstream_repository: XKCP/XKCP
+        upstream_branch: master
+        target_branch: stirling-labs
+        git_pull_args: '--unshallow --allow-unrelated-histories'
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Create new commit
+      if: steps.sync.outputs.has_new_commits
+      run: |
+        git config --global user.email "daemon@stirlinglabs.com"
+        git config --global user.name "StirlingLabsDaemon"
+        echo 'New changes Upstream XKCP' > xkcp.md
+        git add -A
+        git commit -m "XKCP/XKCP main branch has changes"
+    - name: Create Pull Request
+      if: steps.sync.outputs.has_new_commits
+      uses: peter-evans/create-pull-request@v3
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        title: 'XKCP/XKCP main branch has changes'
+        branch: check-xkcp-upstream
+        body: |
+          - Please check changes upstream XKCP master branch.
+        labels: |
+          automated-pr
```

Comment:
Is this going to create a PR in the XKCP repo or the XKCP.NET repo?
It looks like it creates it here in XKCP, which we don't want?

Reply:
@ilkerhalil this should be in a separate PR (and a separate workflow), to avoid confusion.